Sources/swift-format/Frontend/Frontend.swift

```diff
@@ -11,10 +11,11 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
-import SwiftFormat
 import SwiftSyntax
 import SwiftParser
 
+@_spi(Internal) import SwiftFormat
```

Comment:
Reordered imports, but we should probably just setup formatter in #610. I'm happy to do it, just say the word ;)